### PR TITLE
enhancement: mobile beta UI update for activity details

### LINKED ISSUE #3204: [Task]: Update transaction view for mobile beta
### Task description

This view is intended to present detailed information about a transaction. The presentation should be adaptive to the respective asset (IOTA, Shimmer, Assembly, Native Token, etc.)

### Requirements

- [ ] #3205
- [x] #3206

### Acceptance criteria

Can be closed when all above requirements are complete.

### Creation checklist

- [x] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### LINKED ISSUE #3205: [Subtask]: Adapt the layout of the UI overhaul to transaction view
Transactions related to different assets (IOTA, Shimmer, Assembly, etc) shall be displayed adaptively.

### PR DESCRIPTION
**Pull request by paul-boegelsack**
## Summary
Updated the activity details UI to the new mobile beta layout 

### Changelog
```
- Separated desktop and mobile markup (Mobile ActivityDetail)
- Deleted the message id (Mobile ActivityDetail)
- Created a button to view the transaction with the explorer (Mobile ActivityDetail)
- Updated locals
```

## Relevant Issues
Closes #3204
Closes #3205

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [x] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [x] MacOS
	- [ ] Linux
	- [ ] Windows
- __Mobile__
	- [x] iOS
	- [x] Android

### Instructions
Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/components/ActivityDetail.svelte [hunk 1]
```diff
@@ -132,118 +132,200 @@
...
+                        </div>
+                        <Text type="pre" classes="mb-2 mt-0">
+                            {#if receiverAddressesYou[idx]}({receiverAddressesYou[idx].alias}){/if}
+                        </Text>
+                    {/each}
+                </div>
+            {/if}
+            <div class="mb-5 flex justify-center">
+                <button
+                    class="mobile-explorer-button action p-3 w-full text-center rounded-lg font-semibold text-14 bg-white text-blue-500"
```
Comment (amadeu2) [inline packages/shared/components/ActivityDetail.svelte:214]:
```suggestion
                    class="mobile-explorer-button action p-3 w-full text-center rounded-lg font-semibold text-14 bg-white dark:bg-gray-800 text-blue-500"
```
Maybe is lacking dark theme background? This change matches the light theme imo, what do you think?

**Review by amadeu2 — COMMENTED** [review]
The `view in explorer` button should be a bit wider according to Figma design. In general looks good, just minimal changes.

File: packages/shared/components/ActivityDetail.svelte (hunk 1 above)
Comment (paul-boegelsack) [reply packages/shared/components/ActivityDetail.svelte:214]: Yeah, it is, I alway forget about that, thank you 😅

**Review by amadeu2 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**